fh-messaging/lib/fhsrv.js

```diff
@@ -201,49 +203,62 @@ var msg = (function() {
       });
       return;
     }
-    //end request
-    res.json({"message":"accepted messages"});
 
     // first filter out any messages we already have
+    console.log("-----messages: ", messages);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)